neptune/internal/streams/stdstream_uploader.py

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (c) 2019, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import sys
+
+from neptune.internal.streams.channel_writer import ChannelWriter
+
+
+class StdStreamUpload(object):
+
+    def __init__(self, experiment, channel_name, stream):
+        # pylint:disable=protected-access
+        self._channel = experiment._get_channel(channel_name, 'text')
+        self._channel_writer = ChannelWriter(experiment, channel_name)
+        self._stream = stream
+
+    def write(self, data):
+        self._stream.write(data)
+        self._channel_writer.write(data)
+
+    def flush(self):
+        pass
+
+
+class StdOutWithUpload(StdStreamUpload):
+
+    def __init__(self, experiment):
+        super(StdOutWithUpload, self).__init__(experiment, 'stdout', sys.__stdout__)
+        sys.stdout = self
```

Comment:
I'm not sure if this trick will capture `stdout` and `stderr` output from external libraries that import `sys` on their own. Please check how Python import system works.

Reply:
There are a few cases when it won't work, e.g. when library uses stdout through variable assigned before this `stdout` swap but I still think that this solution should be fine in most cases.